docusaurus/docs/dev-docs/migration/v4-to-v5/breaking-changes.md

```diff
@@ -18,6 +18,7 @@ This page is part of the [Strapi v4 to v5 migration](/dev-docs/migration/v4-to-v
 
 * [Some `env`-only configuration options are handled by the server configuration](/dev-docs/migration/v4-to-v5/breaking-changes/removed-support-for-some-env-options)
 * [Configuration filenames should meet strict requirements](/dev-docs/migration/v4-to-v5/breaking-changes/strict-requirements-config-files)
+* [Strapi Server log level configuration](/dev-docs/migration/v4-to-v5/breaking-changes/server-default-log-level)
```

Comment:
Please update this to a sentence that already shortly explains what happens, like we did for the other breaking changes. For instance: "The default server log level is ‘info’". Same for the h1 (line 14 of the breaking change file).